infrastructure/main.tf

```diff
@@ -27,107 +26,33 @@ locals {
-  asp_name                     = (var.asp_name == "use_shared") ? local.sharedAppServicePlan : var.asp_name
-  asp_rg                       = (var.asp_rg == "use_shared") ? local.sharedASPResourceGroup : var.asp_rg
-  website_local_cache_sizeinmb = 1600
-  enable_ase                   = false
-
-  app_settings = {
-    POSTGRES_HOST     = module.db.host_name
-    POSTGRES_PORT     = module.db.postgresql_listen_port
-    POSTGRES_DATABASE = module.db.postgresql_database
-    POSTGRES_USER     = module.db.user_name
-    POSTGRES_PASSWORD = module.db.postgresql_password
-    FORCE_APPLY       = "true"
-
-    # JAVA_OPTS = "${var.java_opts}"
-    # SERVER_PORT = "8080"
-
-    # db
-    SPRING_DATASOURCE_URL      = "jdbc:postgresql://${module.db.host_name}:${module.db.postgresql_listen_port}/${module.db.postgresql_database}?sslmode=require"
-    SPRING_DATASOURCE_USERNAME = module.db.user_name
-    SPRING_DATASOURCE_PASSWORD = module.db.postgresql_password
-
-    MAX_FILE_SIZE = "${var.max_file_size_in_mb}MB"
-
-    # idam
-    IDAM_USER_BASE_URI = var.idam_api_url
-    IDAM_S2S_BASE_URI  = "http://${var.s2s_url}-${local.local_env}.service.core-compute-${local.local_env}.internal"
-
-    # logging vars & healthcheck
-    REFORM_SERVICE_NAME = local.app_full_name
-    REFORM_TEAM         = var.team_name
-    REFORM_SERVICE_TYPE = var.app_language
-    REFORM_ENVIRONMENT  = var.env
-
-    PACKAGES_NAME        = local.app_full_name
-    PACKAGES_PROJECT     = var.team_name
-    PACKAGES_ENVIRONMENT = var.env
-
-    JSON_CONSOLE_PRETTY_PRINT = var.json_console_pretty_print
-    LOG_OUTPUT                = var.log_output
-
-    # addtional log
-    ROOT_LOGGING_LEVEL   = var.root_logging_level
-    LOG_LEVEL_SPRING_WEB = var.log_level_spring_web
-    LOG_LEVEL_DM         = var.log_level_dm
-    SHOW_SQL             = var.show_sql
-
-
-    ENABLE_DB_MIGRATE = "false"
-
-    # Toggles
-    ENABLE_IDAM_HEALTH_CHECK            = var.enable_idam_healthcheck
-    ENABLE_METADATA_SEARCH              = var.enable_metadata_search
-    ENABLE_DOCUMENT_AND_METADATA_UPLOAD = var.enable_document_and_metadata_upload
-    ENABLE_FOLDER_API                   = var.enable_folder_api
-    ENABLE_DELETE                       = var.enable_delete
-    ENABLE_TTL                          = var.enable_ttl
-    ENABLE_THUMBNAIL                    = var.enable_thumbnail
-    ENABLE_TESTING                      = var.enable_testing
-
-    # Document Storage
-    STORAGEACCOUNT_PRIMARY_CONNECTION_STRING   = data.azurerm_key_vault_secret.dm_store_storageaccount_primary_connection_string.value
-    STORAGEACCOUNT_SECONDARY_CONNECTION_STRING = data.azurerm_key_vault_secret.dm_store_storageaccount_secondary_connection_string.value
-    STORAGE_CONTAINER_DOCUMENT_CONTAINER_NAME  = azurerm_storage_container.document_container.name
-
-    TEMP_FORCE_REDEPLOY = "1"
-
-    TASK_ENV = "documentTaskLock-${var.env}"
-  }
+resource "azurerm_resource_group" "rg" {
```

Comment:
you don't need this resource group either - nothing get's deployed there

Reply:
removed.